ff/handlers/bcc_aromaticity.py

```diff
@@ -25,6 +26,18 @@ def __init__(self, *args, smirks: str, **kwargs):
         super(InvalidSmirksError, self).__init__(*args, **kwargs)
         self.smirks = smirks
 
+@contextmanager
+def openeye_log_wrapper():
```

Comment:
I'm pretty unfamiliar with OEChem's context manager and logging this, where does this end up going?

Reply:
Its the same behavior as previously, its dumping it to an oestream and returning it as a context manager. Just allowed me to capture the output the same way in two places without having to check a boolean return value